examples/server.go

```diff
@@ -1,11 +1,12 @@
 package main
 
 import (
-	"github.com/graarh/golang-socketio"
-	"github.com/graarh/golang-socketio/transport"
 	"log"
 	"net/http"
 	"time"
+
+	"github.com/paxosglobal/golang-socketio"
```

Comment:
hmm these import changes are a little awkward... we wouldn't need to do this if we had Go modules set up, but I think that is probably a follow-up.

Reply:
yup they are only needed if we want to run this project alone but not needed with go modules